woocommerce/woocommerce-ios
# IPP Reader Manuals consolidation

### Description
This PR wraps up the `In-Person Payments: Reader Manuals Consolidation` project ( p91TBi-7SX-p2#comment-10794 ) that moves all the manuals to a single consolidated screen, rather than having each manual on a different row, in order to simplify the IPP settings screen.

All changes were made and merged behind the `.consolidatedCardReaderManuals` feature flag. This PR contains:
- [x] Feature flag switch to `true`
- [x] Remove previous code that renders each Card Reader Manual row individually
- [x] Update 9.7 release notes

### Testing instructions
- Select a store that has enabled In-Person Payments.
- Go to Settings > In-Person payments
- See only one row: `Card Reader manuals`. Tap on it:
-- See Manuals for Chipper and M2 in the US store.
-- See the Manual for Wisepad 3 in the CA store.
-- Tap on each Manual to open a web view.

### Screenshots
![RMC 2022-07-11 filter by country](https://user-images.githubusercontent.com/3812076/178489333-8a16690b-a207-42c5-b2d5-728b03e229e1.gif)

| Task  | Resolved? | Issue |
|---|---|---|
| Add a feature flag |  ✅   |  https://github.com/woocommerce/woocommerce-ios/pull/7184 |
| Add Reader Manuals section to IPP menu |  ✅    | https://github.com/woocommerce/woocommerce-ios/pull/7199 |
| CardReadersView: new view for the list of Card Readers - SwiftUI | ✅   | https://github.com/woocommerce/woocommerce-ios/pull/7199 |
| Implement WebViews for each manual |  ✅    | https://github.com/woocommerce/woocommerce-ios/pull/7199 |
| Progress bar for web view |  ✅    | https://github.com/woocommerce/woocommerce-ios/pull/7199 |
| Manual visibility/availability based on each Store/Country |  ✅  | https://github.com/woocommerce/woocommerce-ios/pull/7266 |
| Refactor to use existing SwiftUI components | ✅   | https://github.com/woocommerce/woocommerce-ios/pull/7227 |
| Unit Tests | ✅ | https://github.com/woocommerce/woocommerce-ios/pull/7266 |
| NavigationRow improvement | ✅  | https://github.com/woocommerce/woocommerce-ios/pull/7276 |


---
